pkg/api/customization/monitor/project_graph_action.go

```diff
@@ -69,8 +77,15 @@ func (h *ProjectGraphHandler) QuerySeriesAction(actionName string, action *types
 		if inputParser.Input.MetricParams["namespace"] == "" {
 			return fmt.Errorf("no namespace found")
 		}
-
-		svcName, svcNamespace, svcPort = monitorutil.IstioPrometheusEndpoint()
+		project, err := project.GetSystemProject(clusterName, h.projectLister)
+		if err != nil {
+			return err
+		}
+		app, err := h.appLister.Get(project.Name, monitorutil.IstioAppName)
+		if err != nil {
+			return err
+		}
+		svcName, svcNamespace, svcPort = monitorutil.IstioPrometheusEndpoint(app.Spec.Answers)
```

Comment:
We can use `access-prometheus.cattle-prometheus:80` directly in project level. You need to follow the logic of accessing project level monitoring graph.
At lease you need to use `prometheus-operated.cattle-prometheus:9090` endpoint with bearer token in the header.